src/main/java/com/trashit/back/entity/User.java

```diff
@@ -66,6 +68,9 @@
     @JoinColumn(name = "role_id")
     private Role role;
 
+    @OneToMany(mappedBy = "user")
+    private Collection<Score> scores = new ArrayList<Score>();
```

Comment:
```suggestion
    private Collection<Score> scores = new ArrayList<>();
```
C'est pas utile de ce coté là